newsfragments/1259.feature

```diff
@@ -0,0 +1 @@
+Add an option to show lattice(s) in the crystal rather than laboratory frame.
```

Comment:
```suggestion
dials.reciprocal_lattice_viewer: Add an option to show lattice(s) in the crystal rather than laboratory frame.
```

confusing otherwise